python/ray/worker.py

```diff
@@ -2293,6 +2369,13 @@ def disconnect(worker=global_worker):
     worker.cached_remote_functions_and_actors = []
     worker.serialization_context_map.clear()
 
+    subprocess.call(
```

Comment:
This should be part of https://github.com/ray-project/ray/blob/46351957bb633b1860b4e379e592e2cd9455600a/python/ray/services.py#L170-L188